.travis.yml

```diff
@@ -25,25 +25,25 @@ jobs:
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.8
-        - NP_BUILD_DEP="numpy==1.19.1"
-        - NP_TEST_DEP="numpy==1.19.1"
+        - NP_BUILD_DEP="numpy==1.19"
+        - NP_TEST_DEP="numpy==1.19"
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.9
-        - NP_BUILD_DEP="numpy==1.19.1"
-        - NP_TEST_DEP="numpy==1.19.1"
```

Comment:
Actually, can you try numpy 1.19.5? We build on the oldest supported numpy platform since numpy's ABI is only forward compatible. 

Bumping here would break users running numpy 1.19 on aarch64 I think.